### ISSUE TITLE
Deprecated feature used

### ISSUE THREAD

**Issue by grv87** [chef-cookbooks/jenkins#569]
This won't work in Chef 13:
```
==> default: Deprecated features used!
==> default:   Chef::Platform.set is deprecated at 4 locations:
==> default:     - /var/chef/cache/cookbooks/jenkins/libraries/plugin.rb:443:in `<top (required)>'
==> default:     - /var/chef/cache/cookbooks/jenkins/libraries/slave.rb:417:in `<top (required)>'
==> default:     - /tmp/vagrant-cache/chef/cookbooks/jenkins/libraries/slave.rb:417:in `<top (required)>'
==> default:     - /var/chef/cache/cookbooks/jenkins/libraries/user.rb:186:in `<top (required)>'
==> default:    See https://docs.chef.io/deprecations_chef_platform_methods.html for further details.
```

**Comment by iennae** [comment]
Thanks for reporting. We are in the process of a big overhaul of all the community cookbooks to fix all the Chef 13 deprecation issues.

**Comment by tas50** [comment]
I’m adding the Type: Jump In GitHub label to this issue. This is a great issue for someone to get their feet wet with and we’d love a PR to resolves the issue.

**Comment by mrala** [comment]
This issue can be closed since it was resolved in #558.

**Comment by iennae** [comment]
@mrala Thanks for letting us know!
